presto-docs/src/main/sphinx/functions/window.rst

```diff
@@ -58,6 +58,10 @@ The window definition has 3 components:
   the current row. The sort key must either be the same type of ``expression`` or can be coerced to the
   same type as ``expression``.
 
+  In ``GROUPS`` mode, frame starts and ends of ``expression PRECEDING`` or ``expression FOLLOWING``
```

Comment:
Its not clear what a group represents. Can you explain what a group means? A query example below will also be good to add

Reply:
Add explanation and also examples to demonstrate the three types of frame definitions.